src/templates/partials/nft-cards.njk

```diff
@@ -1,4 +1,28 @@
-<img class="nft-card__image" src="src\images\static\nft-card-cat.webp" alt="NFT Item 2">
-    <span class="nft-card__badge nft-card__badge--new">NEW</span>
-        <p class="nft-card__price">$199</p>
-        <h3 class="nft-card__title">NAME PRODUCT</h3> 
\ No newline at end of file
+<section>
+    <div>
+      <h2>Catalog</h2>
+      <div>
+        <a href="#">SEE ALL</a>
+        <div>
+          <button>&#8592;</button>
+          <button>&#8594;</button>
+        </div>
+      </div>
+    </div>
+    <div>
+      <ul>
+        {% for nft in nftCards %}
+        <li>
+          <a href="#">
```

Comment:
Create new file for card - `nft-card.njk` for example, and use include here.
Creating of template for card is better, because you can reuse it for catalog page markup later.